### PR TITLE
Gives the pickaxe a proper description.

### PR DESCRIPTION
**Pull request by Al-1ce**
## About The Pull Request

Pickaxes. My scourge. My bane. My ruination. These god-damnable little things, staring at me from their cathedra's upon my desk. They know if they fall I have no chance of returning them to their proper place, no recourse. Their usage and utility long exiled. Before the smooth white polymer shell I stand a frail and broken woman. My mind whimpers at their sight. I think I no longer possess the proper lucidity today to truly express what they have done to me, not anymore. How can a broken mind self-diagnose?

## Why It's Good For The Game

The pickaxe has been given a proper description relative to the current political climate in 2470.

## Changelog

:cl:
fix: Pickaxes are given a proper description.
/:cl:

### CODE REVIEW

File: code/game/objects/items/tools/mining_tools.dm [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 /obj/item/tool/pickaxe
 	name = "pickaxe"
+	desc =  "Don't give the homeless pickaxes and adderal and tell them theres gold undearneath our most important road systems."
```
Comment (novaepee) [inline code/game/objects/items/tools/mining_tools.dm:8]: there's instead of theres

**Comment by Goblinu** [comment]
I TAKE THE ROLE OF A SILLY CLOWN

**Comment by Xander3359** [comment]
I have never truly witnessed a perfect PR, before today. I really must say, this PR has changed my life, before this PR, I was simply a lost traveler going through life. But now, my eyes are open, now I can see the truth. This is truly what the TGMC needs, it really represents what the essence of UNGA is. 

Maintainers, speedmerge this ASAP.

**Comment by Raven-Industries** [comment]
is "undearneath" intentional?

**Comment by Al-1ce** [comment]
> is "undearneath" intentional?

No, it was not.

**Comment by TiviPlus** [comment]
I dont see the funny and this isnt a proper description

**Comment by Al-1ce** [comment]
> I dont see the funny and this isnt a proper description

I think you lack a sense of humor and waltz around with a stick up your ass all day.

**Comment by Al-1ce** [comment]
> I dont see the funny and this isnt a proper description

* I think you lack a sense of humor
* You may have a stick up your ass
* Proper ..how, exactly? There are dozens of items which have been recently added that have goofy/silly descriptions, not to mention swathes of old stuff (1-2 years) that's stuck around for ages, with nobody seeming to mind or give a shit.

It's a meme PR anyways but you're shitting me if you think we actually have standards for .defs here. Do not make me bring out the comically large tweezers.

**Pull request closed without merging**